styles.css

```diff
@@ -0,0 +1,227 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700;800&display=swap");
+
+:root {
+  --inter: "Inter", sans-serif;
+  --crete: "Crete Round", serif;
+  --dark-background: #1f2937;
+  --hero-color: #f9faf8;
+  --hero-sec: #e5e7eb;
+  --button-color: #3882f6;
+  --info-text: #1f2937;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+a {
+  text-decoration: none;
+}
+
+.logo {
+  color: white;
+  text-decoration: none;
+  width: 80px;
+}
+
+.logo > img {
+  width: 80%;
+  padding-top: 20px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 5rem 0 5rem;
+  height: 3rem;
+  padding-top: 30px;
+  /* padding-bottom: 50px; */
+  background-color: var(--dark-background);
+  font-family: var(--inter);
+}
+
+.nav-icons {
+  display: flex;
+}
+
+.nav-icons > li {
+  margin-left: 2rem;
+}
+li {
+  list-style: none;
+}
+
+.nav-icons a {
+  color: var(--hero-sec);
+}
+
+.headline {
+  display: flex;
+  background-color: var(--dark-background);
+  column-gap: 1rem;
+  padding: 100px;
+  align-items: center;
+}
+
+.headline > div {
+  flex: 1;
+  height: fit-content;
+}
+.headline-image {
+  width: 200px;
+  /* height: 16rem; */
+  /* overflow: hidden; */
+}
+.headline-image > img {
+  width: 100%;
+}
+
+section {
+  padding: 0 5rem 0 5rem;
+}
```

Comment:
- [ ] This `padding: 0 5rem 0 5rem;` on the `section` selector has been overridden by the padding declared in the `.headline` class selector due to specificity. you can remove one of them.